crates/driver/src/arguments.rs

```diff
@@ -259,6 +259,12 @@ pub struct Arguments {
     /// ZeroEx API key.
     #[clap(long, env)]
     pub zeroex_api_key: Option<String>,
+
+    /// The number of decimal places considered when comparing the objective value of different solutions.
+    /// Note, that objective values are computed in wei. E.g. a value of -15 would consider solutions with
+    /// with objective value 0.0012 ETH and 0.0016 ETH equivalent.
+    #[clap(long, env, default_value = "0")]
+    pub solution_comparison_decimal_precision: i32,
```

Comment:
The way this is specified is unintuitive for me because there are negative values. It feels more natural for this to be a positive number. With the current implementation this would be "ignore the N least significant digits in base 10".

This could be more general too in order to not restrict us to powers of 10. The definition could be "round/floor objective value to a multiple of N". Then you could set it to 1 in order to not change anything, 10 to ignore the least significant digit, 100 to ignore the first two digits, but also something like 5 to round in between. Not sure if we need it this fine grained.

Reply:
I agree with you. I keep going back and forth over the naming here. The thing with calling it "precision" is that a larger would intuitively imply a **more precise** evaluation. I will change the semantics to how you suggested it and call it `solution_comparison_decimal_cutoff` but would be greatful for a better name.


> The definition could be "round/floor objective value to a multiple of N

I think this is overkill for now.

Reply:
Intuitively (at least to me), a "precision of 10" means that we compare the first 10 decimal places of the value (i.e., your example of 0.0012 ETH and 0.0016 ETH would be 3 and not -15 for me). I also agree that negative precision values are a bit weird and suggest renaming.

That being said, this isn't super critical for the PR IMO.